package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dfinity/nns",
-  "version": "0.2.2",
+  "version": "nightly-2022-02-28",
```

Comment:
Should the version be commited? I would say we keep the official minor-major version in the repo spontaneously but just my two cents.

Reply:
No, this is an accident.  Thanks!